app/containers/MessageActions/index.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable complexity */
```

Comment:
I don't think it's interesting to hide this "problem".
The warning is there for a reason, ignore it and it is not the best solution in most cases, especially since there is not even a TODO specifying that we should resolve this problem later.